op-supervisor/supervisor/backend/db/update.go

```diff
@@ -50,6 +57,20 @@ func (db *ChainsDB) initializedSealBlock(chain eth.ChainID, block eth.BlockRef)
 	return nil
 }
 
+func (db *ChainsDB) isInteropActive(timestamp uint64) bool {
+	if db.depSet == nil {
+		return true
+	}
```

Comment:
I don't have context so could easily be wrong, but this seems unsafe - shortly if we don't have a dependency set interop would be inactive?